pandas/tests/indexing/test_loc.py

```diff
@@ -1644,6 +1644,18 @@ def test_loc_setitem_mask_td64_series_value(self):
         assert expected == result
         tm.assert_frame_equal(df, df_copy)
 
+    def test_loc_setitem_boolean_list(self):
```

Comment:
can you parameterize the rhs by array as well as list

Reply:
ahh i see the comment, but still think can parameterize.

Reply:
Done